components/chart/MultiBarSleepChart.tsx

```
@@ -0,0 +1,90 @@
+            sleep.push(record.duration.asMinutes());
+            offsetData.push({ start: startTime, stop: record.endTime, sameDay });
+            labels.push(record.endTime);
+        }
+
+        for (const record of offsetData) {
+            const startHourMinutes = record.start.hour() * 60 + record.start.minute() + (record.sameDay ? 24 * 60 : 0);
+            offset.push(startHourMinutes - earliestHoursMinutes);
+        }
+
+        // Group records by day
+        let groupRowIndex: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)